presto-main/src/test/java/com/facebook/presto/operator/aggregation/reservoirsample/TestUnweightedDoubleReservoirSample.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.operator.aggregation.reservoirsample;
+
+import org.testng.annotations.Test;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertTrue;
+import static org.testng.Assert.fail;
+
+public class TestUnweightedDoubleReservoirSample
+{
+    @Test
+    public void testGetters()
+    {
+        final UnweightedDoubleReservoirSample sample =
+                new UnweightedDoubleReservoirSample(200);
+
+        assertEquals(sample.getMaxSamples(), 200);
+    }
+
+    @Test
+    public void testIllegalBucketCount()
+    {
+        try {
+            new UnweightedDoubleReservoirSample(0);
+            fail("exception expected");
+        }
+        catch (IllegalArgumentException e) {
+        }
+    }
+
+    @Test
+    public void testFew()
+    {
+        final UnweightedDoubleReservoirSample sample =
+                new UnweightedDoubleReservoirSample(200);
+
+        sample.add(1.0);
+        sample.add(2.0);
+        sample.add(3.0);
+
+        Set<Double> samples = new HashSet<>();
```

Comment:
`assertEquals(sample.stream().boxed().collect(toImmutableSet()), ImmutableSet.of(1.0, 2.0, 3.0));`

Reply:
The stream interface doesn't exist anymore.

I'm using 
```
    @Test
    public void testFew()
    {
        UnweightedDoubleReservoirSample reservoir = new UnweightedDoubleReservoirSample(200);

        reservoir.add(1.0);
        reservoir.add(2.0);
        reservoir.add(3.0);

        assertEquals(
                Arrays.asList(reservoir.getSamples()).stream().collect(toImmutableSet()),
                ImmutableSet.of(1.0, 2.0, 3.0));
    }
```

but am getting 

```
java.lang.AssertionError: Sets differ: expected [1.0, 2.0, 3.0] but got [[D@66d1af89]
Expected :1.0, 2.0, 3.0
Actual   :[D@66d1af89
```

Could you please advise? Thanks!

Reply:
@atavory Make it

```
        assertEquals(
                Arrays.stream(reservoir.getSamples()).boxed().collect(toImmutableSet()),
                ImmutableSet.of(1.0, 2.0, 3.0));
```

Reply:
@atavory Or better yet `assertEquals(reservoir.getSamples(), new double[] {1.0, 2.0, 3.0});`